blog/_posts/2023-09-06-client-quadlet.md

```diff
@@ -0,0 +1,193 @@
+    apiVersion: v1
+    kind: PersistentVolumeClaim
+    metadata:
+      name: ghost-pv-claim
+      labels:
+        app: ghost
+    spec:
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: 20Gi
+    ---
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: inlets-ghost-demo
+    spec:
+      containers:
+      - name: ghost
+        image: docker.io/library/ghost:5.59.0-alpine
+        env:
+        - name: url
+          value: https://{{ inlets_server_domain }}
+        - name: NODE_ENV
+          value: development
+        volumeMounts:
+        - name: ghost-persistent-storage
+          mountPath: /var/lib/ghost/content
+        resources:
+          requests:
+            memory: "64Mi"
+            cpu: "250m"
+          limits:
+            memory: "128Mi"
+            cpu: "500m"
+      - name: inlets
+        image: ghcr.io/inlets/inlets-pro:{{ inlets_version | default('0.9.21') }}
+        args:
+        - "http"
+        - "client"
+        - "--url=wss://{{ inlets_server_ip }}:8123"
+        - "--token-file=/var/secrets/inlets-token/inlets-token"
+        - "--license-file=/var/secrets/inlets-license/inlets-license"
+        - "--upstream=http://127.0.0.1:2368"
+        volumeMounts:
+        - mountPath: /var/secrets/inlets-token
+          name: inlets-token
+        - mountPath: /var/secrets/inlets-license
+          name: inlets-license
+        resources:
+          requests:
+            memory: "64Mi"
+            cpu: "250m"
+          limits:
+            memory: "128Mi"
+            cpu: "500m"
+      volumes:
+      - name: ghost-persistent-storage
+        persistentVolumeClaim:
+          claimName: ghost-pv-claim
+      - name: inlets-license
+        secret:
+          secretName: inlets-license
+      - name: inlets-token
+        secret:
+          secretName: inlets-token
+    ```
+2. Create the `kube_data.json` file
+    ```json
+    {
+        "inlets_server_ip": "192.168.1.1",
+        "inlets_server_domain": "ghost.example.com"
+    }
+    ```
+3. Generate the `inlets-ghost.yaml` file
+    ```bash
+    j2 inlets-ghost.yml.j2 kube_data.json > inlets-ghost.yml
+    ```
+
+#### Quadlet `inlets-ghost.kube` file
+```
+[Kube]
+Yaml=inlets-ghost.yml
+```
+
+#### Create and run the Quadlet
+Since the client does not require any root privilages, it can be executed as a user service
+
+1. Copy the `inlets-ghost.yaml` and `inlets-ghost.kube` files to  `~/.config/containers/systemd/`
+    ``` bash
+    sudo cp inlets-ghost.yaml inlets-ghost.kube ~/.config/containers/systemd/
+    ```
+2. Reload the systemd daemon
+    ```bash
+    systemctl --user daemon-reload
+    ```
+3. Start the service
+    ```bash
+    systemctl --user start inlets-ghost.service
+    ```
+
+## Access the blog
+
+Once all resources are up and running you can access you blog at your at `https://< inlets_server_domain >`
+
+To create your admin account, add `/ghost` to the end of the URL.
```

Comment:
Could you add a conclusion, and further links to Qualet examples and / or ideas you may give the reader of what they may want to expose or try next, after what they've learned here?

Reply:
I've added a `What`s next` section providing some links on Quadlet. Anything else you think I should/can add?